pkg/cli/dump_test.go

```diff
@@ -38,6 +38,13 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
 )
 
+// sleepOffset performs a time.Sleep for COCKROACH_MAX_OFFSET. This is used
```

Comment:
i'm not sure why this is sufficient - who says the backwards time jump is bounded by this value?